org-formation/organization.yaml

```diff
@@ -445,3 +445,20 @@ Organization:
       Alias: org-sagebase-sceptredev
       Tags:
         <<: !Include ./_default_org_tags.yaml
+
+
+  #---------- Personal User Accounts  -----------
+  hxo89cta7a0e:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-hxo89cta7a0e
+      RootEmail: aws-hxo89cta7a0e@sagebase.org
+      Alias: org-sagebase-hxo89cta7a0e
+      Tags:
+        <<: !Include ./_default_org_tags.yaml
+        Department: TBD
+        Project: TBD
+        CostCenter: TBD
```

Comment:
Instead of "TBD" can we use a valid Cost Center, like 000000 NO PROGRAM?

Reply:
We are just waiting for @tthyer to review and provide a valid values for these tags.

Reply:
Tess won't be reviewing these for awhile, so let's just leave this PR open until she's back from leave.